packages/js/components/src/woo-product-section-item/woo-product-section-item.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Slot, Fill } from '@wordpress/components';
+import { createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { createOrderedChildren, sortFillsByOrder } from '../utils';
+
+type WooProductSectionItemProps = {
+	id: string;
+	location: string;
+	pluginId: string;
+	order?: number;
+};
+
+type WooProductFieldSlotProps = {
+	location: string;
+};
+
+export const WooProductSectionItem: React.FC< WooProductSectionItemProps > & {
+	Slot: React.FC< Slot.Props & WooProductFieldSlotProps >;
+} = ( { children, order = 1, location } ) => (
```

Comment:
One last suggestion, we should change the default order to `10` or maybe even `20`.
That way it leaves more room to add specific orders for our own sections and fields.

Reply:
Good thought, I'll do this as part of the details PR so I can get this merged.